packages/common-components/src/Toasts/types.ts

```diff
@@ -0,0 +1,22 @@
+
+export type ToastType = "success" | "error"
+
+export type ToastPosition = "topLeft" | "topRight" | "bottomLeft" | "bottomRight"
+export interface Toast {
+  id: string
+  type: ToastType
+  title: string
+  subtitle?: string
+  progress?: number
+  onProgressCancel?(): void
```

Comment:
use `() => void` notation
```suggestion
  onProgressCancel?: () => void
```